lib/src/install/completion.rs

```diff
@@ -245,7 +245,7 @@ pub(crate) async fn run_from_anaconda(rootfs: &Dir) -> Result<()> {
 
 /// From ostree-rs-ext, run through the rest of bootc install functionality
 pub async fn run_from_ostree(rootfs: &Dir, sysroot: &Utf8Path, stateroot: &str) -> Result<()> {
-    crate::cli::require_root()?;
+    crate::cli::require_root(false)?;
```

Comment:
Non-blocking: One larger cleanup I think we could do is have an `Environment` struct that gathers global state like this (uid, whether we have cap_sys_admin, whether `/run/ostree-booted` exists, whether we appear to be in a container), and then query requirements on it.

e.g. here we're passing `false` for `is_container` but we don't actually know that's false or not.

Not a blocker, just an optional followup.